pennylane_lightning_gpu/lightning_gpu.py

```diff
@@ -19,6 +19,8 @@
 from warnings import warn
 from itertools import product
 
+#from mpi4py import MPI
```

Comment:
You can likely wrap this in a try-except clause and use it to set a boolean variable to indicate if it was successful. That could then be used to import the MPI-backed other classes later, if it was a success.

Reply:
As discussed, I just remove `mpi4py` here to keep the previous workflow for the single gpu backend.